tools/filters/headWrapper.xml

```diff
@@ -6,20 +6,13 @@
   <requirements>
     <requirement type="package" version="8.31">coreutils</requirement>
   </requirements>
-  <stdio>
-    <exit_code range="1:140"/>
```

Comment:
What is exit code 141?

Reply:
See e.g. https://stackoverflow.com/questions/19120263/why-exit-code-141-with-grep-q

Reply:
Short summary @bernt-matthias: the pipe you've implemented with *head* at the consuming end is **expected** to have the sender, *tail* or *cat* to fail with exit code 141. That's why this shouldn't be treated as an error state.
The problem for .eu though is: while the 141 gets produced on test.galaxyproject.org, on usegalaxy.org and on our own server when executing the job script manually, when usegalaxy.eu runs it, it sees exit code 1 and marks the job as failed.
We have no idea why (a suspect is cgroups, but that's just a guess because there's no better explanation), nor what we could do about it, but removing the pipe as suggested here would give us back a functional tool.